chapter-01-polynomes.tex

```diff
@@ -433,21 +433,21 @@ \section{Divisibilité et racines}
 \end{displaymath}
 tel que $\deg(r(x)) < \deg(g(x))$. On  obtient alors
 \begin{displaymath}
-  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x). 
+  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x) = q_0(x)g(x) + r(x), \;\text{où} q_0(x) = q(x) + \frac{a_n}{ b_m } x^{n-m}. 
 \end{displaymath}
 %
-Supposons maintenant qu'il existe deux autres polynômes $q'(x)$ et $r'(x)$ tels que 
+Supposons maintenant qu'il existe deux autres polynômes $q'(x) \neq q_0(x)$ et $r'(x) \neq r(x)$ tels que 
```

Comment:
Also here, try to stick to the names in the theorem. $q(x)$ and $r(x)$ repsectively.

Reply:
I only called it q_0(x) since it seems to me that q(x) is taken already (when we invoke the induction hypothesis).

Reply:
See above